manual_test_nodejs/README.md

```diff
@@ -12,3 +12,10 @@ In the plugin directory:
 `node subprocess.js`
 
 It should stop after 10 seconds with the proper halting message
+
+## Testing resource proxy
+
+`npm start -- --resourceRoutes`
```

Comment:
please add `resourceRoutes` to the list of cli arguments on top of the docs.

Reply:
This is not in the main README, but in the manual tests, so does this still apply?

Reply:
oups I did not see it. No it does not :p This is in the right place. Is `resourceRoute` in the main readme ?

Reply:
Yes it is. Resolved.